test/fixtures/cookbooks/languages_rust/recipes/default.rb

```diff
@@ -1,8 +1,8 @@
 include_recipe 'chef-sugar::default'
 include_recipe 'languages::default'
 
-package 'curl' unless mac_os_x?
+package 'curl' unless mac_os_x? || windows?
```

Comment:
How do we feel moving the execution of this package resource as part of provider? Sort of like we've done for required gems in other resources/providers we've written (e.g. `artifactory_*`).